sps30-i2c/sps30_example_usage.c

```diff
@@ -54,6 +54,23 @@ int main(void) {
     }
     printf("SPS sensor probing successful\n");
 
+    uint8_t fw_major;
+    uint8_t fw_minor;
+    ret = sps30_read_firmware_version(&fw_major, &fw_minor);
+    if (ret) {
```

Comment:
Sidenote, for a follow-up change: for sps30_start measurement() and sps30_read_measurement(), we're checking for ```ret < 0``` instead. Since this is the example, should we keep those consistent? I'd suggest going with the ```if (ret) {``` everywhere rather than assuming all error codes are negative (since we're just passing on error codes from embedded-common in the case of errors), or even ```if (ret != NO_ERROR) {``` if we want to be overly verbose. Thoughts?

Reply:
This is true for almost all instances, but sometimes the return value gets (ab-)used for returning a positive value *or* a negative error. But I'd rather get rid of these cases and always use the return value for errors. Then I'd also use a better naming of `error` so one can just do:
```
if (error) {
...
}
```
Which is much clearer IMO.

Reply:
agreed, that form would be the best